xbout/tests/test_animate.py

```diff
@@ -58,3 +60,228 @@ def test_animate1D(self, create_test_file):
         animation = ds['n'].isel(y=2, z=0).bout.animate1D(save_as="%s/test" % save_dir)
 
         assert isinstance(animation, Line)
+
+    def test_animate_list(self, create_test_file):
+
+        save_dir, ds = create_test_file
+
+        animations = ds.isel(z=3).bout.animate_list(['n', ds['T'].isel(x=2),
+                                                     ds['n'].isel(y=1, z=2)])
+
+        assert isinstance(animations.blocks[0], Pcolormesh)
+        assert isinstance(animations.blocks[1], Pcolormesh)
+        assert isinstance(animations.blocks[2], Line)
+
+    def test_animate_list_1d_default(self, create_test_file):
+
+        save_dir, ds = create_test_file
+
+        animations = ds.isel(y=2, z=3).bout.animate_list(['n', ds['T'].isel(x=2),
+                                                          ds['n'].isel(y=1, z=2)])
+
+        assert isinstance(animations.blocks[0], Line)
+        assert isinstance(animations.blocks[1], Pcolormesh)
+        assert isinstance(animations.blocks[2], Line)
+
+    def test_animate_list_animate_over(self, create_test_file):
+
+        save_dir, ds = create_test_file
+
+        animations = ds.isel(z=3).bout.animate_list(['n', ds['T'].isel(t=2),
+                                                     ds['n'].isel(y=1, z=2)],
+                                                    animate_over='x')
+
+        assert isinstance(animations.blocks[0], Pcolormesh)
+        assert isinstance(animations.blocks[1], Pcolormesh)
+        assert isinstance(animations.blocks[2], Line)
+
+    def test_animate_list_save_as(self, create_test_file):
+
+        save_dir, ds = create_test_file
+
+        animations = ds.isel(z=3).bout.animate_list(['n', ds['T'].isel(x=2),
+                                                     ds['n'].isel(y=1, z=2)],
+                                                    save_as="%s/test" % save_dir)
+
+        assert isinstance(animations.blocks[0], Pcolormesh)
+        assert isinstance(animations.blocks[1], Pcolormesh)
+        assert isinstance(animations.blocks[2], Line)
+
+    def test_animate_list_fps(self, create_test_file):
```

Comment:
This isn't testing that the fps was actually set. Should do 
```python
assert animations.fps == fps
```

Reply:
(also to be really nitpicky the animatplot logic is that you have created one animation (singular) which consists of multiple blocks (plural))